### PR TITLE
Features/slider

### PR DESCRIPTION
**Pull request by almalmalm**
Add dinamic data to trailers slider. Implement api route.

### CODE REVIEW

**Review by NurkenInc — APPROVED** [review]
Looks good to me, but it's better to resolve build fails!

**Review by zinuu4 — APPROVED** [review]
Well done!)

File: src/shared/ui/mute-button/ui.tsx
Comment (zinuu4) [inline src/shared/ui/mute-button/ui.tsx:1]: It's better to unite mute button and volume button into one and change icon inside depending on coming props

File: src/shared/ui/volume-button/ui.tsx
Comment (zinuu4) [inline src/shared/ui/volume-button/ui.tsx:1]: It's better to unite mute button and volume button into one and change icon inside depending on coming props

**Review by zinuu4 — CHANGES_REQUESTED** [review]

File: src/shared/ui/mute-button/ui.tsx
Comment (almalmalm) [reply src/shared/ui/mute-button/ui.tsx:1]: Delete mute button component.

File: src/shared/ui/volume-button/ui.tsx
Comment (almalmalm) [reply src/shared/ui/volume-button/ui.tsx:1]: Add switch volume logic to  volume button component.

**Review by zinuu4 — APPROVED** [review]
Well done!

**Pull request merged**